src/iden3comm/utils/did.ts

```diff
@@ -37,39 +24,26 @@ export const resolveDIDDocument = async (
   }
 };
 
-export const getDIDComponentById = (
+export const resolveVerificationMethods = (
```

Comment:
Are you sure that reduce function works correctly? Theoretically `reduce` function is used to reduce array to one element... But verification method should return list of vms...